data/assets/01.js

```diff
@@ -0,0 +1,23 @@
+// data stale warning mechanism
```

Comment:
Is it possible to implement in Dash code to avoid this JS code?

Reply:
You can embed the javascript directly in  dash-python code but I don't think that's the way we want to go

Reply:
Embedding JS in `.py` files wouldn't be an improvement.

Reply:
Done